roboteam_ai/src/world/FutureWorld.cpp

```diff
@@ -0,0 +1,36 @@
+//
+// Created by thijs on 19-3-19.
+//
+
+#include "FutureWorld.h"
+#include "History.h"
+
+#include <roboteam_ai/src/utilities/Constants.h>
+
+namespace rtt {
+namespace ai {
+namespace world {
+
+void FutureWorld::updateFutureBall(Ball &ball, double time) {
```

Comment:
I would like the names of these functions to make it clear it is linear extrapolation